fastq_filtrator.py

```diff
@@ -0,0 +1,64 @@
+def gc(read, gc_bounds=(0, 100)): #the function for GC filtering
```

Comment:
Название функции не отражает сути. Не понятно, что мы делаем с этим `gc`. Расчитываем `gc-состав` в процентах или кол-во `gc` в штуках или проверяем на соответствие границам
```suggestion
def check_gc_content(read, gc_bounds=(0, 100)): #the function for GC filtering
```

Reply:
По названию функции должно быть понятно, что она делает